scripts/run_bitcoin.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ev
+
+BITCOIND_VERSION=${BITCOIN_VERSION:-0.18.1}
```

Comment:
q: Are we using an old version intentionally?

Reply:
Yeah, bitcoind made a breaking change in the `GetBlockchainInfo` API, so we can't run 0.7 with the newer versions of core.